omnigibson/utils/sampling_utils.py

```diff
@@ -20,6 +20,7 @@
 # Create settings for this module
 m = create_module_macros(module_path=__file__)
 
+m.DEBUG_SAMPLING = False
```

Comment:
I think the gm.DEBUG flag should strictly override this since that's a global flag. So maybe False OR gm.DEBUG?

Reply:
The problem with that is that this flag has a LOT of cases where it will entirely *stop* the execution of code and enable breakpoints, markers, etc. so it actually affects execution. The regular DEBUG flag seems to mostly be used for logging? So these should be separate.